src/nodejs/lambda-handlers/submission.js

```diff
@@ -144,8 +144,17 @@ async function submitMethod(event, user) {
 async function reviewMethod(event, user) {
   const { id, approve } = event;
   const status = await db.submission.getState({ id });
+  const stepType = status.step.type;
+  let eventType;
```

Comment:
Can be simplified using ternary operator:
```
const eventType = !approve ? 'review_rejected' : stepType === 'review' ? 'review_approved' : 'workflow_promote_step';
```
I think this approach is more straightforward for devs who are more comfortable with Node; however, it does create more confusion for those not as familiar with ternary operators or nesting ternary operators. Feel free to change if you'd like, but this comment's more for knowledge share than a change request 👍

Reply:
I considered writing it like that but as a node transplant that line is decently confusing to me.

Reply:
I thought about writing it like that but as a node transplant that line is harder for me to read but I do see how if you are very comfortable with ternary operators it would be cleaner.